python/orca/src/bigdl/orca/learn/pytorch/training_operator.py

```diff
@@ -276,8 +276,6 @@ def train_batch(self, batch, batch_info):
                 output = self.model(features)
             elif isinstance(features, dict):
                 output = self.model(**features)
-            elif isinstance(features, (tuple, list)):
-                output = self.model(*features)
```

Comment:
remove this in `forward_batch` as well

Reply:
I think this is not right here, we should remove the case when features is dict instead of tuple right? @Chaselzxy